lua/onedarkpro/highlights/plugins/telescope.lua

```diff
@@ -16,8 +16,14 @@ function M.groups(theme)
             bg = config.cursorline,
             fg = theme.palette.gray,
         },
-        TelescopeNormal = { fg = theme.palette.fg },
-        TelescopeBorder = { fg = theme.palette.gray },
+        TelescopeNormal = {
+            bg = config.options.transparency and "NONE" or theme.generated.float_bg,
+            fg = theme.palette.fg,
+        },
+        TelescopeBorder = {
+            bg = config.options.transparency and "NONE" or theme.generated.float_bg,
+            fg = theme.palette.gray,
+        },
```

Comment:
Note that this is a change to the theme's default Telescope window background color--even when `window_unfocused_color` is not enabled.

I believe that previously Telescope inherited the background color from the normal window background color, and hence used `theme.palette.pg`. I think that `float_bg`, now that we have it defined, is more appropriate for Telescope and more consistent with the background color used for other floating windows for completion, hovering definitions, etc..

We can switch these to `bg = config.options.transparency and "NONE" or theme.palette.bg,` if you prefer to only fix the color when `window_unfocused_color` without changing it in the process.